aspnetcore/security/authorization/simple.md

```diff
@@ -66,7 +66,9 @@ public class AccountController : Controller
 This would allow only authenticated users to the `AccountController`, except for the `Login` action, which is accessible by everyone, regardless of their authenticated or unauthenticated / anonymous status.
 
 > [!WARNING]
-> `[AllowAnonymous]` bypasses all authorization statements. If you combine `[AllowAnonymous]` and any `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level, any `[Authorize]` attributes on the same controller (or on any action within it) are ignored.
+> `[AllowAnonymous]` bypasses authorization statements. If you combine `[AllowAnonymous]` and an `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level:
+> * Any `[Authorize]` attributes on the same controller or action methods on the controller, the authorization requirements specified by `[Authorize]` ware ignored.
+> * Any related authentication is still be done.
```

Comment:
Would change to: Authentication middleware is not short-circuited but doesn't need to succeed.